lib/CircuitBreaker.js

```diff
@@ -43,13 +43,13 @@ function updateCircuitBreakerConfigForImplicitOutputQueue(cbConf, groupId, topic
 const circuitBreakerGauge = ZenkoMetrics.createGauge({
     name: 's3_circuit_breaker',
     help: 'Circuit Breaker State',
-    labelNames: ['type'],
+    labelNames: ['type', 'promErr'],
 });
 
 function startCircuitBreakerMetricsExport(cb, cbType, intervalMs = collectDefaultMetricsIntervalMs) {
     const type = cbType || 'generic';
     setInterval(() => {
-        circuitBreakerGauge.set({ type }, cb.state);
+        circuitBreakerGauge.set({ type, promErr: cb.promErr }, cb.state);
```

Comment:
this is actually not correct. there is only "time serie" for each set of labels, and it does not expire.
this is not really an issue for counters (i.e. the number of "events" which happened), but a gauge represents the current number of something, and must thus be reset when that number drops...

here, looking over time:

```
# T0 : circuit breaker ok
s3_circuit_breaker{type="xxx", promErr="false"} 0

# T1 : circuit breaker reported an error (still state 0)
s3_circuit_breaker{type="xxx", promErr="false"} 0  # ← still present !
s3_circuit_breaker{type="xxx", promErr="true"}  0  # ← set by the timer

# T1 : circuit breaker still in error, state update to 2
s3_circuit_breaker{type="xxx", promErr="false"} 0  # ← still present !
s3_circuit_breaker{type="xxx", promErr="true"}  2  # ← set by the timer

# T2 : circuit breaker back to normal (no error, state 0)
s3_circuit_breaker{type="xxx", promErr="false"} 0  # ← set by the timer
s3_circuit_breaker{type="xxx", promErr="true"}  2  # ← still present !
```

Reply:
I went for a counter : https://github.com/scality/backbeat/pull/2516/commits/73b8545ac169e6e00dc3e1ce637db01656c487f2#diff-07825dc8c7adf67b6bbea289468e3446d7646ec1c8f5b9c361b381807fb2ce28R58-R60
and the alert is now triggered based on the increase : https://github.com/scality/backbeat/pull/2516/commits/73b8545ac169e6e00dc3e1ce637db01656c487f2#diff-794b759f3f971e96a7be74227bbf5af68323e43bfb5dbbe2bc66fbe5dfb5f521R342